package.json

```diff
@@ -63,7 +63,7 @@
     "postinstall": "rimraf ./src/**/*.scss.d.ts",
     "service-status": "node scripts/serviceStatus.js",
     "start": "cross-env NODE_ENV=development NETWORK=private node --max_old_space_size=4096 ./node_modules/webpack-dev-server/bin/webpack-dev-server.js --config webpack.dev.config.js",
-    "start-dev-profiling": "cross-env NODE_ENV=dev-profiling NETWORK=rinkeby API_URL=https://daostack-alchemy.herokuapp.com S3_BUCKET=daostack-alchemy-staging webpack-dev-server --config webpack.dev.config.js",
+    "start-dev-profiling": "cross-env NODE_ENV=dev-profiling NETWORK=rinkeby webpack-dev-server --config webpack.dev.config.js",
```

Comment:
why this change ?

Reply:
we dont use API_URL anymore

Reply:
is that related to this pr ?

Reply:
well no. but it was such a small thing i bundled it in :)